curated_transformers/models/bert/config.py

```diff
@@ -36,17 +38,20 @@ class BertAttentionConfig:
     hidden_width: int
     num_attention_heads: int
     dropout_prob: float
+    qkv_mode: QkvMode
```

Comment:
I don't think we should at this to the attention config until a need arises. This is really a property of the architecture and we should just hardcode it where the attention layer is constructed.

Reply:
Removed.